test/dotnet-list-package.Tests/GivenDotnetListPackage.cs

```diff
@@ -349,5 +349,51 @@ public void ItRunsInCurrentDirectoryWithPoundInPath()
                 .Should()
                 .Pass();
         }
+
+        [Fact]
+        public void ItRecognizesRelativePathsForAProject()
```

Comment:
We also need an equivalent test on the NuGet side as well. 

The issue here is integration between the 2 components/repos, so we should add tests in both repos.

Reply:
What would be testing in the NuGet side though? Currently, the contract is that dotnet SDK should always call NuGet with an absolute path. NuGet is not expected to recognize relative paths.

Reply:
We have tests that integrate a version of the .NET SDK with NuGet.
https://github.com/NuGet/NuGet.Client/tree/dev/test/NuGet.Core.FuncTests/Dotnet.Integration.Test downloads a version of the SDK and patches on top of that, effectively testing NuGet & .NET.

Reply:
https://github.com/NuGet/Home/issues/13238 Added a tracking issue